WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -225,8 +225,9 @@ def post_results
         body += "#{record_strs.join(", ")}.  \n" # Trailing spaces for markdown give us a <br>
       end
     end
-    post = Post.new(title: title, body: body, author: current_user, world_readable: true)
-    post_post(post)
+    comp.update(results_posted_at: Time.now)
+    comp.save!
```

Comment:
The page I looked at didn't list `update!()`, but I agree that that's better.

Two caveats:

**Small:** If the update were to throw an exception, wouldn't the user get an ugly exception page? As written, the line can never fail unless the schema changes, so maybe that's OK.

**Big:** Despite the name, `CompetitionsController.post_results()` doesn't create results. That's done earlier somewhere. Wouldn't it make more sense to update `results_posted_at` when the results are actually posted?